API/Handlers/Implementations/OAuthHandler.cs

```diff
@@ -172,7 +172,7 @@ IUserRepository userRepository
             accessToken = GenerateAccessToken(
                 user.Id.ToString(),
                 _jwtAudience,
-                user.Scopes,
+                [.. user.Scopes, "user"],
```

Comment:
Surprised this syntax works, didn't even know this was a thing.

Reply:
Yea me neither lol... I found it because the formatter automatically formatted `client.Scopes.Append("client").ToArray();` to `[.. client.Scopes, "client"];` so I guess we have to use it :P